packages/sdk/src/events/turn-tracker.ts

```
@@ -0,0 +1,235 @@
+      return {
+        errorType: 'network_error',
+        errorMessage: 'Network connection failed'
+      }
+    }
+    
+    return {
+      errorType: 'unknown_error',
+      errorMessage: error.message
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)